packages/common/src/utils/cookies.ts

```diff
@@ -0,0 +1,6 @@
+export const readCookie = (name: string): string => {
```

Comment:
was using state in the host app initially - am thinking that we would likely have display language tied to a user, as per desktop and mobile; in which case the language is set prior to loading the app. have implemented as a cookie to simulate that approach and keep the state a little cleaner. Easy enough to have it dynamic if we want to though 🤷‍♂️